spec/models/form526_submission_spec.rb

```diff
@@ -28,7 +28,7 @@
   let(:backup_submitted_claim_status) { nil }
 
   before do
-    Flipper.disable(:disability_compensation_production_tester)
```

Comment:
Did you mean to remove change the name of the flipper here?

Reply:
Yes. The `:disability_compensation_production_tester` flipper does not need to be disabled in this section of the spec, and the `:validate_saved_claims_with_json_schemer` does.

Notably -- and this PR is still in draft, so it's something I will tackle before officially publishing it -- this method of disabling Flippers is now deprecated. [The flipper docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample) stipulate that the new paradigm going forward is to use `allow(Flipper).to receive(:enabled?).with(:name_of_flag).and_return(true/false)`.